.buildkite/bootstrap-cmake.sh

```diff
@@ -7,10 +7,12 @@ source /etc/profile.d/enable-gcc-toolset.sh
 VESPA_CMAKE_SANITIZERS_OPTION=""
 if [[ $VESPA_USE_SANITIZER != null ]]; then
     VESPA_CMAKE_SANITIZERS_OPTION="-DVESPA_USE_SANITIZER=$VESPA_USE_SANITIZER"
+    VESPA_CMAKE_CCACHE_OPTION="-DVESPA_USE_CCACHE=true"
```

Comment:
I would have thought this should be false, but maybe I am misunderstanding how the flag is used?

Reply:
This should be false. Thanks 👍 Fixed in new commit.